### PR TITLE
[Keyboard] Add labbeminiv1

### PR DESCRIPTION
**Pull request by alabbe9545**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/labbe/labbeminiv1/config.h [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1BBE
```
Comment (drashna) [inline keyboards/labbe/labbeminiv1/config.h:21]: This vendor ID is in use already.  Please change it. 

https://www.the-sz.com/products/usbid/index.php?v=0x1BBE&p=&n=

tHe further past 0x4000, the better, actually.

File: keyboards/labbe/labbeminiv1/keymaps/rgb/keymap.c [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  [0] = LAYOUT_wasd( KC_W, KC_A, KC_S, KC_D),
+};
+
+
+/*void matrix_init_user(void) {
+    // Set RGB to rainbow mood light
+    rgblight_enable();
+    rgblight_sethsv(255,255,255);
+    rgblight_mode(6);
+}*/
```
Comment (drashna) [inline keyboards/labbe/labbeminiv1/keymaps/rgb/keymap.c:28]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by alabbe9545** [comment]
Fixed both issues

**Comment by alabbe9545** [comment]
Is there an ETA for this?

**Review by drashna — APPROVED** [review]

File: keyboards/labbe/labbeminiv1/rules.mk [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no       	# Mouse keys
+EXTRAKEY_ENABLE = no       	# Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = no  	    # Enable keyboard RGB underglow
```
Comment (fauxpark) [inline keyboards/labbe/labbeminiv1/rules.mk:18]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by alabbe9545** [comment]
the change was applied

**Comment by alabbe9545** [comment]
Is there any ETA for this?

**Review by drashna — APPROVED** [review]

**Comment by alabbe9545** [comment]
Hello im waiting for a review,
Please add label "awaiting review"
Thanks.

File: keyboards/labbe/labbeminiv1/keymaps/rgb-matrix/config.h [hunk 4]
```diff
@@ -0,0 +1,26 @@
+/*
```
Comment (fauxpark) [inline keyboards/labbe/labbeminiv1/keymaps/rgb-matrix/config.h:1]: Keymap folders should not contain `-`.

**Comment by alabbe9545** [comment]
Hello @fauxpark, '-' removed from keymap folder

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**